cms/models.py

```diff
@@ -228,11 +228,40 @@ class CoursesInProgramPage(CourseProgramChildPage):
         blank=True,
         null=True,
     )
+    override_contents = models.BooleanField(
+        blank=True,
+        default=False,
+        help_text="Manually select contents below. Otherwise displays all courses related associated with the program.",
```

Comment:
Is that check will just display only courses in a program ?

Reply:
This check allows us to define `contents` of the carousel. The contents can be either `CoursePage` or `ProgramPage` pages, and the `contents` don't have to be homogeneous, meaning we can select both programs and courses to be displayed in the carousel, in the order that we select them.